src/compose/ComposeBox.js

```diff
@@ -221,6 +223,15 @@ class ComposeBox extends PureComponent<Props, State> {
     this.setState(({ isMenuExpanded }) => ({
       isMenuExpanded: !isMenuExpanded,
     }));
+    setTimeout(
```

Comment:
Instead of using `setTimeout` here, we should pass a callback into `start()` in `AnimatedComponent`.